lib/libc/sys/coaccept.c

```diff
@@ -47,11 +47,11 @@ coaccept(void * __capability * __capability cookiep,
 
 	if (outbuf != NULL) {
 		assert(__builtin_is_aligned(outbuf, sizeof(void * __capability)));
-		assert(__builtin_is_aligned(outlen, sizeof(void * __capability)));
+		//assert(__builtin_is_aligned(outlen, sizeof(void * __capability)));
```

Comment:
These can't be committed as commented-out code (and certainly not with the style-violating C-style comments)